lib/filesystem_utils.pm

```diff
@@ -509,4 +510,33 @@ sub validate_lsblk {
     return $errors;
 }
 
+=head2
+
+Generate xfstests subtests list
+This function translate test range to single tests, with xfstests test name format.
+Input an parameter with list of subtest name, Return a hash of test list.
+e.g. generate "xfs/001-003,xfs/005" into "{xfs/001 => 1, xfs/002 => 1, xfs/003 => 1, xfs/005 => 1}"
+
+=cut
+
+sub generate_xfstests_list {
+    my $raw_list = shift;
```

Comment:
i am not familiar with those modules but i think it would be sensible to add a check that $raw_list is passed and print a die if not. wdyt?